lib/globus_client.rb

```diff
@@ -12,6 +12,7 @@
 
 # Client for interacting with the Globus API
 class GlobusClient # rubocop:disable Metrics/ClassLength
+  include Errors
```

Comment:
My take here is that it's customary generally to have custom exceptions declared within the top-level namespace of a gem.